rockcraft.yaml

```diff
@@ -4,7 +4,11 @@ description: Spark ROCK
 license: Apache-2.0
 
 version: "3.4.2"
+# version:spark:3.4.2
```

Comment:
Do we want to keep those commented lines?

Reply:
We need. I mean, I thought that we have "3.4.2" which is the "charmed" version. We have then the versions of the single components: spark and jupyter, in this case.

It is a product/strategy decision that the charmed one corresponds to the version of one (the main) product